css/fonts.css

```diff
@@ -0,0 +1,7 @@
+@font-face {
+  font-family: 'PT Serif';
+  src: url('.fonts/ptserif-bold-webfont.woff2') format('woff2'), 
+       url('.fonts/ptserif-bold-webfont.woff') format('woff');
```

Comment:
Codacy found an issue: [Expected indentation of 4 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/html_abrmhah/pullRequest?prid=6295173)